src/main/java/seedu/address/model/lesson/MakeUpLesson.java

```diff
@@ -12,7 +12,7 @@
     /**
      * Every field must be present and not null.
      *
-     * @param date Start and end Date of lesson.
+     * @param date           Start and end Date of lesson.
```

Comment:
want to adjust this back to the normal unaligned ver looks v out of place here 🥲